docs/src/pages/components/tables/EnhancedTable.tsx

```diff
@@ -227,17 +228,7 @@ const useStyles = makeStyles((theme: Theme) =>
     table: {
       minWidth: 750,
     },
-    visuallyHidden: {
-      border: 0,
-      clip: 'rect(0 0 0 0)',
-      height: 1,
-      margin: -1,
-      overflow: 'hidden',
-      padding: 0,
-      position: 'absolute',
-      top: 20,
-      width: 1,
-    },
+    visuallyHidden,
```

Comment:
```
-- top: 20,	
++ whiteSpace: 'nowrap',
```
Are are ending up with slightly different styles.

Reply:
Good that we have normalized then :)